includes/class-wc-payments-token-service.php

```diff
@@ -365,6 +370,41 @@ public function get_account_saved_payment_methods_list_item_link( $item, $paymen
 		return $item;
 	}
 
+	/**
+	 * Controls the output for Wallet tokens on the My account page.
+	 *
+	 * @param  array                                        $item          Individual list item from woocommerce_saved_payment_methods_list.
+	 * @param  WC_Payment_Token|WC_Payment_Token_WCPay_Link $payment_token The payment token associated with this method entry.
+	 * @return array                                        Filtered item
+	 */
+	public function get_account_saved_payment_methods_list_item_wallet( $item, $payment_token ) {
+		if ( 'cc' !== strtolower( $payment_token->get_type() ) ) {
+			return $item;
+		}
+
+		$wallet_type = $payment_token->get_meta( '_wcpay_wallet_type', true );
+
+		if ( empty( $wallet_type ) ) {
+			return $item;
+		}
+
+		// Google Pay and Apple Pay are separate payment methods, so we can retrieve them from the registered payment methods class.
+		$payment_method = WC_Payments::get_payment_method_by_id( $wallet_type );
+		if ( ! $payment_method || ! method_exists( $payment_method, 'get_title' ) ) {
+			return $item;
+		}
+
+		$original_brand          = $item['method']['brand'] ?? '';
+		$item['method']['brand'] = sprintf(
+			/* translators: 1: wallet name, 2: card brand */
+			_x( '%1$s %2$s', 'Payment token with wallet', 'woocommerce-payments' ),
+			$payment_method->get_title(),
+			$original_brand
+		);
+
+		return $item;
+	}
```

Comment:
Ensures that we display whether the payment token comes from Google Pay/Apple Pay (on the "My account" page) when a payment token is saved.

<img width="856" height="435" alt="Image" src="https://github.com/user-attachments/assets/5d480af4-b196-415f-8398-61006bda4f84" />

<img width="1091" height="851" alt="Image" src="https://github.com/user-attachments/assets/2a1d6034-e30e-4c7d-8800-3e4536a65459" />

I considered doing something like `Visa (Google Pay) ending in 4242`. Unfortunately, WC Core uses `ucwords`. So the output looked like `Visa (google Pay) ending in 4242`. And I didn't think that using find/replace was going to be a robust solution.